src/components/10-atoms/checkbox/ui.test.js

```diff
@@ -127,6 +110,18 @@ test('should set refId on label and input', async t => {
     .ok();
 });
 
+fixture('Checkbox - Disabled and checked').page(
```

Comment:
Why did you moved that test?

Reply:
because the test above need the path from the top